sql/xsql-shell/src/main/scala/org/apache/spark/sql/xsql/shell/SparkXSQLShell.scala

```diff
@@ -108,7 +108,8 @@ object SparkXSQLShell extends Logging {
       sqlContext = spark.sqlContext
       conf = spark.sparkContext.getConf
       sc = spark.sparkContext
-      processInitFile(conf)
+      // we shouldn't load .hiverc twice
+      // processInitFile(conf)
```

Comment:
`processInitFile` is only used for load `.hiverc`?

Reply:
> `processInitFile` is only used for load `.hiverc`?

Yes

Reply:
I think we should change coment as 'Don't call `processInitFile` again to avoid load `.hiverc` twice.'